controllers/namespace_controller.go

```diff
@@ -76,16 +77,30 @@ func (r *NamespaceReconciler) reconcile(ctx context.Context, ns *corev1.Namespac
 func (r *NamespaceReconciler) propagateMeta(ctx context.Context, ns, parent *corev1.Namespace) error {
 	orig := ns.DeepCopy()
 	for _, key := range r.LabelKeys {
```

Comment:
Would you refactor the logic so that it would look like:

```go
for k, v := range parent.Labels {
    if r.matchLabelKey(k) {
        ns.Labels[k] = v
    }
}
```

Reply:
Thanks.
fixed:
https://github.com/cybozu-go/accurate/pull/8/commits/dd7cf644f4fa7fa5a7d1c164888a97130fb3963a